src/react-components/ui-root.js

```diff
@@ -1484,10 +1485,11 @@ class UIRoot extends Component {
                   scene={this.props.scene}
                 />
               )}
-            <RoomLayout
+            <RoomLayoutContainer
+              scene={this.props.scene}
+              store={this.props.store}
               objectFocused={!!this.props.selectedObject}
               streaming={streaming}
-              onResizeViewport={this.onResizeViewport}
```

Comment:
You removed the reference to `this.onResizeViewport` here, but the method still exists and is now unused.